recipes/extract_model/meta.yaml

```diff
@@ -1,50 +1,39 @@
 {% set name = "extract_model" %}
-{% set version = "0.1.1" %}
+{% set version = "0.1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/axiom-data-science/extract_model/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 5257438da53c8f568f6d21eab798ea8725fae905d78c84b6870fc0ee84ab1961
-  # sha256 is the preferred checksum -- you can get it for a file with:
-  #  `openssl sha256 <file name>`.
-  # You may need the openssl package, available on conda-forge:
-  #  `conda install openssl -c conda-forge``
+  sha256: e702fbfdd7ff04f3a5fb1fb23e0c6cbb1bd2c2c2861d91ccad6e46d01a1ae70f
 
 build:
-  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
-  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
-  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
-  # noarch: python
   number: 0
-  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
-  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
-  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{version}}
   script: "{{ PYTHON }} -m pip install . -vv"
-  skip: true  # [win]
 
 requirements:
   host:
-    - python ==3.8
+    - python >=3.8,<3.9
     - pip
     - setuptools-scm
   run:
+    - python >=3.8,<3.9
     - cf_xarray
     - cmocean
     - dask
-    - esmf =8.1.1=mpi_openmpi_h852a670_0  # [linux64]
-    - esmpy =8.1.1=mpi_openmpi_py38h9ab444c_0  # [linux64]
-    - esmf =8.1.1=mpi_openmpi_h047fafc_0  # [osx]
-    - esmpy =8.1.1=mpi_openmpi_py38hf504846_0  # [osx]
```

Comment:
Forcing mpi like that is not recommended and is a bit brittle. We should allow for the end user to define the variant they need to install. I'm guessing it would work with both mpi and nompi, right?

Reply:
I think xESMF will work with different numbered versions of ESMF and ESMPy but only the MPI compilation for each operating system. Without MPI the code won't actually run (at least on Mac, the docs for xESMF imply that it might not be true for all operating systems). Is there a way to generalize this?

Reply:
I am working on some tests to see about this and the versioning may not need to be so strict anymore!! I will post back here when I have it better figured out.

Reply:
It is odd that the serial version won't run. If that is true then we need a way to generalize one mpi :-/
I'll ask around if that is possible. Note that one of the reasons to not do what you have above is to allow end users to use custom mpi setups like those described in https://conda-forge.org/docs/user/tipsandtricks.html#using-external-message-passing-interface-mpi-libraries